modin/core/storage_formats/pandas/parsers.py

```diff
@@ -123,8 +123,14 @@ def find_common_type_cat(types):
     """
     if all(isinstance(t, pandas.CategoricalDtype) for t in types):
         if all(t.ordered for t in types):
+            categories = np.unique([c for t in types for c in t.categories])[0]
+            if categories.ndim != 0:
+                categories = np.sort(categories)
+            else:
+                # Cannot use np.sort() on array with ndim == 0
+                categories = [categories.tolist()]
```

Comment:
This seems like it will create a 2d list, is that what's expected?

Reply:
In this `else` case, categories in a 0-dimensional numpy array so `tolist()` just converts it to a normal int. Then the `[]` will ensure we pass an actual list to `pandas.CategoricalDtype`.

Reply:
Ok this makes sense, can we add that as a comment above this line? Reading `.tolist` to me implies that it will return a `list` object.

Reply:
I can also change it to .item() which would probably be much clearer.